ci/release/update-version.sh

```diff
@@ -59,8 +59,12 @@ for FILE in dependencies.yaml conda/environments/*.yaml; do
 done
 
 # Update ucx-py version
+# TODO: Remove "ci/*" file lines after Jenkins scripts are removed
 sed_runner "s/export UCX_PY_VERSION=.*/export UCX_PY_VERSION='${NEXT_UCX_PY_VERSION}'/g" ci/checks/style.sh
 sed_runner "s/export UCX_PY_VERSION=.*/export UCX_PY_VERSION='${NEXT_UCX_PY_VERSION}'/g" ci/cpu/build.sh
 sed_runner "s/export UCX_PY_VERSION=.*/export UCX_PY_VERSION='${NEXT_UCX_PY_VERSION}'/g" ci/gpu/build.sh
```

Comment:
```suggestion
```
On second thought, Jenkins will now source this variable from `conda_build_config.yaml` too. So these lines can be removed now.

Reply:
I was leaving this category of change for “post-Jenkins cleanup.” There are tons of similar actions to take, like removing the actual UCX version definitions from Jenkins. Happy to see that kind of thing done in this PR if you wish, but I have been skipping those changes for now.